ERROR - 2024-05-31 00:26:04 --> mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT jam_kerj...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT jam_kerj...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT jam_kerj...')
#3 C:\xampp\htdocs\wiharta\app\Models\Laporan_model.php(311): CodeIgniter\Database\BaseConnection->query('SELECT jam_kerj...')
#4 C:\xampp\htdocs\wiharta\app\Controllers\Laporan.php(149): App\Models\Laporan_model->Dataabsen(Array)
#5 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\Laporan->Tabelabsenlaporanbulanan()
#6 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Laporan))
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-05-31 00:26:04 --> Call to a member function getRowArray() on bool
in APPPATH\Models\Laporan_model.php on line 311.
 1 APPPATH\Controllers\Laporan.php(149): App\Models\Laporan_model->Dataabsen([...])
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Laporan->Tabelabsenlaporanbulanan()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Laporan))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-31 00:26:04 --> filesize(): stat failed for C:\xampp\htdocs\wiharta\writable\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 161.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'filesize(): stat failed for C:\\xampp\\htdocs\\wiharta\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6', 'SYSTEMPATH\\Session\\Handlers\\FileHandler.php', 161)
 2 SYSTEMPATH\Session\Handlers\FileHandler.php(161): filesize('ROOTPATH\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6')
 3 [internal function]: CodeIgniter\Session\Handlers\FileHandler->read('vnclrq1pc8t080stalnbbnpd7vc2mgr6')
 4 SYSTEMPATH\Session\Session.php(912): session_start()
 5 SYSTEMPATH\Session\Session.php(234): CodeIgniter\Session\Session->startSession()
 6 SYSTEMPATH\Config\Services.php(665): CodeIgniter\Session\Session->start()
 7 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
 8 SYSTEMPATH\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 9 SYSTEMPATH\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
10 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
11 SYSTEMPATH\Common.php(931): CodeIgniter\Config\BaseService::__callStatic('session', [])
12 VENDORPATH\myth\auth\src\Authentication\AuthenticationBase.php(128): session('logged_in')
13 VENDORPATH\myth\auth\src\Authentication\LocalAuthenticator.php(67): Myth\Auth\Authentication\AuthenticationBase->isLoggedIn()
14 VENDORPATH\myth\auth\src\Filters\PermissionFilter.php(21): Myth\Auth\Authentication\LocalAuthenticator->check()
15 SYSTEMPATH\Filters\Filters.php(173): Myth\Auth\Filters\PermissionFilter->before(Object(CodeIgniter\HTTP\IncomingRequest), [...])
16 SYSTEMPATH\CodeIgniter.php(455): CodeIgniter\Filters\Filters->run('admin/laporan/tabellaporanbulanan', 'before')
17 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
18 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-31 00:26:04 --> filesize(): stat failed for C:\xampp\htdocs\wiharta\writable\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 161.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'filesize(): stat failed for C:\\xampp\\htdocs\\wiharta\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6', 'SYSTEMPATH\\Session\\Handlers\\FileHandler.php', 161)
 2 SYSTEMPATH\Session\Handlers\FileHandler.php(161): filesize('ROOTPATH\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6')
 3 [internal function]: CodeIgniter\Session\Handlers\FileHandler->read('vnclrq1pc8t080stalnbbnpd7vc2mgr6')
 4 SYSTEMPATH\Session\Session.php(912): session_start()
 5 SYSTEMPATH\Session\Session.php(234): CodeIgniter\Session\Session->startSession()
 6 SYSTEMPATH\Config\Services.php(665): CodeIgniter\Session\Session->start()
 7 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
 8 SYSTEMPATH\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 9 SYSTEMPATH\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
10 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
11 SYSTEMPATH\Common.php(959): CodeIgniter\Config\BaseService::__callStatic('session', [])
12 VENDORPATH\myth\auth\src\Controllers\AuthController.php(29): service('session')
13 SYSTEMPATH\CodeIgniter.php(889): Myth\Auth\Controllers\AuthController->__construct()
14 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
15 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
16 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-31 00:26:04 --> filesize(): stat failed for C:\xampp\htdocs\wiharta\writable\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 161.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'filesize(): stat failed for C:\\xampp\\htdocs\\wiharta\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6', 'SYSTEMPATH\\Session\\Handlers\\FileHandler.php', 161)
 2 SYSTEMPATH\Session\Handlers\FileHandler.php(161): filesize('ROOTPATH\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6')
 3 [internal function]: CodeIgniter\Session\Handlers\FileHandler->read('vnclrq1pc8t080stalnbbnpd7vc2mgr6')
 4 SYSTEMPATH\Session\Session.php(912): session_start()
 5 SYSTEMPATH\Session\Session.php(234): CodeIgniter\Session\Session->startSession()
 6 SYSTEMPATH\Config\Services.php(665): CodeIgniter\Session\Session->start()
 7 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
 8 SYSTEMPATH\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 9 SYSTEMPATH\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
10 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
11 SYSTEMPATH\Common.php(959): CodeIgniter\Config\BaseService::__callStatic('session', [])
12 VENDORPATH\myth\auth\src\Controllers\AuthController.php(29): service('session')
13 SYSTEMPATH\CodeIgniter.php(889): Myth\Auth\Controllers\AuthController->__construct()
14 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
15 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
16 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-31 00:26:04 --> filesize(): stat failed for C:\xampp\htdocs\wiharta\writable\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 161.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'filesize(): stat failed for C:\\xampp\\htdocs\\wiharta\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6', 'SYSTEMPATH\\Session\\Handlers\\FileHandler.php', 161)
 2 SYSTEMPATH\Session\Handlers\FileHandler.php(161): filesize('ROOTPATH\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6')
 3 [internal function]: CodeIgniter\Session\Handlers\FileHandler->read('vnclrq1pc8t080stalnbbnpd7vc2mgr6')
 4 SYSTEMPATH\Session\Session.php(912): session_start()
 5 SYSTEMPATH\Session\Session.php(234): CodeIgniter\Session\Session->startSession()
 6 SYSTEMPATH\Config\Services.php(665): CodeIgniter\Session\Session->start()
 7 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
 8 SYSTEMPATH\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 9 SYSTEMPATH\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
10 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
11 SYSTEMPATH\Common.php(959): CodeIgniter\Config\BaseService::__callStatic('session', [])
12 VENDORPATH\myth\auth\src\Controllers\AuthController.php(29): service('session')
13 SYSTEMPATH\CodeIgniter.php(889): Myth\Auth\Controllers\AuthController->__construct()
14 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
15 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
16 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-31 00:26:04 --> filesize(): stat failed for C:\xampp\htdocs\wiharta\writable\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 161.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'filesize(): stat failed for C:\\xampp\\htdocs\\wiharta\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6', 'SYSTEMPATH\\Session\\Handlers\\FileHandler.php', 161)
 2 SYSTEMPATH\Session\Handlers\FileHandler.php(161): filesize('ROOTPATH\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6')
 3 [internal function]: CodeIgniter\Session\Handlers\FileHandler->read('vnclrq1pc8t080stalnbbnpd7vc2mgr6')
 4 SYSTEMPATH\Session\Session.php(912): session_start()
 5 SYSTEMPATH\Session\Session.php(234): CodeIgniter\Session\Session->startSession()
 6 SYSTEMPATH\Config\Services.php(665): CodeIgniter\Session\Session->start()
 7 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
 8 SYSTEMPATH\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 9 SYSTEMPATH\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
10 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
11 SYSTEMPATH\Common.php(959): CodeIgniter\Config\BaseService::__callStatic('session', [])
12 VENDORPATH\myth\auth\src\Controllers\AuthController.php(29): service('session')
13 SYSTEMPATH\CodeIgniter.php(889): Myth\Auth\Controllers\AuthController->__construct()
14 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
15 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
16 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-31 00:26:04 --> filesize(): stat failed for C:\xampp\htdocs\wiharta\writable\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6
in SYSTEMPATH\Session\Handlers\FileHandler.php on line 161.
 1 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'filesize(): stat failed for C:\\xampp\\htdocs\\wiharta\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6', 'SYSTEMPATH\\Session\\Handlers\\FileHandler.php', 161)
 2 SYSTEMPATH\Session\Handlers\FileHandler.php(161): filesize('ROOTPATH\\writable\\session/ci_sessionvnclrq1pc8t080stalnbbnpd7vc2mgr6')
 3 [internal function]: CodeIgniter\Session\Handlers\FileHandler->read('vnclrq1pc8t080stalnbbnpd7vc2mgr6')
 4 SYSTEMPATH\Session\Session.php(912): session_start()
 5 SYSTEMPATH\Session\Session.php(234): CodeIgniter\Session\Session->startSession()
 6 SYSTEMPATH\Config\Services.php(665): CodeIgniter\Session\Session->start()
 7 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session(Object(Config\App), false)
 8 SYSTEMPATH\Config\BaseService.php(194): CodeIgniter\Config\BaseService::__callStatic('session', [...])
 9 SYSTEMPATH\Config\Services.php(637): CodeIgniter\Config\BaseService::getSharedInstance('session', null)
10 SYSTEMPATH\Config\BaseService.php(253): CodeIgniter\Config\Services::session()
11 SYSTEMPATH\Common.php(959): CodeIgniter\Config\BaseService::__callStatic('session', [])
12 VENDORPATH\myth\auth\src\Controllers\AuthController.php(29): service('session')
13 SYSTEMPATH\CodeIgniter.php(889): Myth\Auth\Controllers\AuthController->__construct()
14 SYSTEMPATH\CodeIgniter.php(473): CodeIgniter\CodeIgniter->createController()
15 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
16 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-31 09:01:27 --> mysqli_sql_exception: Column 'tglinv' in field list is ambiguous in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\Exportmutasi_model.php(242): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Exportmutasi.php(408): App\Models\exim\barangjadi\Exportmutasi_model->count_allexportmutasi()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Exportmutasi->Datatablesexport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Exportmutasi))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2024-05-31 09:09:25 --> mysqli_sql_exception: Column 'tglinv' in field list is ambiguous in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\Exportmutasi_model.php(242): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Exportmutasi.php(408): App\Models\exim\barangjadi\Exportmutasi_model->count_allexportmutasi()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Exportmutasi->Datatablesexport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Exportmutasi))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2024-05-31 09:09:31 --> mysqli_sql_exception: Column 'tglinv' in field list is ambiguous in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\Exportmutasi_model.php(242): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Exportmutasi.php(408): App\Models\exim\barangjadi\Exportmutasi_model->count_allexportmutasi()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Exportmutasi->Datatablesexport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Exportmutasi))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2024-05-31 09:18:15 --> mysqli_sql_exception: Column 'tglinv' in field list is ambiguous in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\Exportmutasi_model.php(242): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Exportmutasi.php(408): App\Models\exim\barangjadi\Exportmutasi_model->count_allexportmutasi()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Exportmutasi->Datatablesexport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Exportmutasi))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2024-05-31 09:18:16 --> mysqli_sql_exception: Column 'tglinv' in field list is ambiguous in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\Exportmutasi_model.php(242): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Exportmutasi.php(408): App\Models\exim\barangjadi\Exportmutasi_model->count_allexportmutasi()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Exportmutasi->Datatablesexport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Exportmutasi))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2024-05-31 09:18:19 --> mysqli_sql_exception: Column 'tglinv' in field list is ambiguous in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\Exportmutasi_model.php(242): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Exportmutasi.php(408): App\Models\exim\barangjadi\Exportmutasi_model->count_allexportmutasi()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Exportmutasi->Datatablesexport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Exportmutasi))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2024-05-31 10:49:08 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('154.100.100.3', 'root', '', 'mps_mrp', 3306, '', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\MutasiProduksi_model.php(124): CodeIgniter\Database\BaseConnection->query('SELECT rec_hdr_...')
#4 C:\xampp\htdocs\wiharta\app\Controllers\exim\MutasiProduksi.php(74): App\Models\exim\barangjadi\MutasiProduksi_model->Aju()
#5 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Mutasiproduksi->Edit()
#6 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Mutasiproduksi))
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\MutasiProduksi_model.php(124): CodeIgniter\Database\BaseConnection->query('SELECT rec_hdr_...')
#3 C:\xampp\htdocs\wiharta\app\Controllers\exim\MutasiProduksi.php(74): App\Models\exim\barangjadi\MutasiProduksi_model->Aju()
#4 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Mutasiproduksi->Edit()
#5 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Mutasiproduksi))
#6 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-05-31 10:49:08 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\exim\barangjadi\MutasiProduksi_model.php(124): CodeIgniter\Database\BaseConnection->query('SELECT rec_hdr_freight_bill as aju FROM rech_hdr where rec_hdr_freight_bill NOT LIKE \'%-%\' AND (tgl_beacukai BETWEEN \'2022-01-01\' AND CURDATE()) GROUP BY rec_hdr_freight_bill ORDER BY rec_hdr_freight_bill')
 3 APPPATH\Controllers\exim\MutasiProduksi.php(74): App\Models\exim\barangjadi\MutasiProduksi_model->Aju()
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\exim\Mutasiproduksi->Edit()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Mutasiproduksi))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-31 13:22:30 --> mysqli_sql_exception: Column 'tglinv' in field list is ambiguous in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\barangjadi\Exportmutasi_model.php(242): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Exportmutasi.php(408): App\Models\exim\barangjadi\Exportmutasi_model->count_allexportmutasi()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Exportmutasi->Datatablesexport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Exportmutasi))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
CRITICAL - 2024-05-31 14:38:41 --> Undefined variable $bulan
in APPPATH\Controllers\Laporan.php on line 153.
 1 APPPATH\Controllers\Laporan.php(153): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $bulan', 'APPPATH\\Controllers\\Laporan.php', 153)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Laporan->Tabelabsenlaporanbulanan()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Laporan))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-05-31 14:59:25 --> Undefined variable $bulan
in APPPATH\Controllers\Laporan.php on line 153.
 1 APPPATH\Controllers\Laporan.php(153): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $bulan', 'APPPATH\\Controllers\\Laporan.php', 153)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Laporan->Tabelabsenlaporanbulanan()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Laporan))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-31 15:08:40 --> mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1669): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(551): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\wiharta\app\Models\exim\import\Import_model.php(124): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Import.php(449): App\Models\exim\import\Import_model->count_filteredimport(Array)
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Import->Datatablesimport()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Import))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}
ERROR - 2024-05-31 17:07:27 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('154.100.100.3', 'root', '', 'mps_mrp', 3306, '', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\wiharta\app\Models\exim\import\Import_model.php(44): CodeIgniter\Database\BaseConnection->query('SELECT item_des...')
#4 C:\xampp\htdocs\wiharta\app\Controllers\exim\Import.php(170): App\Models\exim\import\Import_model->Itemimport('RMIPP00117')
#5 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Import->Saveedit()
#6 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Import))
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\wiharta\app\Models\exim\import\Import_model.php(44): CodeIgniter\Database\BaseConnection->query('SELECT item_des...')
#3 C:\xampp\htdocs\wiharta\app\Controllers\exim\Import.php(170): App\Models\exim\import\Import_model->Itemimport('RMIPP00117')
#4 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Import->Saveedit()
#5 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Import))
#6 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-05-31 17:07:27 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\exim\import\Import_model.php(44): CodeIgniter\Database\BaseConnection->query('SELECT item_description,itembc FROM item_ccn WHERE item LIKE \'%RMIPP00117%\'')
 3 APPPATH\Controllers\exim\Import.php(170): App\Models\exim\import\Import_model->Itemimport('RMIPP00117')
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\exim\Import->Saveedit()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Import))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2024-05-31 17:17:21 --> Error connecting to the database: mysqli_sql_exception: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('154.100.100.3', 'root', '', 'mps_mrp', 3306, '', 0)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `beacuka...', Array, false)
#4 C:\xampp\htdocs\wiharta\system\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\wiharta\app\Models\exim\import\Import_model.php(181): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\wiharta\app\Controllers\exim\Import.php(473): App\Models\exim\import\Import_model->get_datatablesimportjkt(Array)
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Import->Datatablesimportjkt()
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Import))
#9 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond in C:\xampp\htdocs\wiharta\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\wiharta\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\wiharta\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `beacuka...', Array, false)
#3 C:\xampp\htdocs\wiharta\system\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\wiharta\app\Models\exim\import\Import_model.php(181): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\wiharta\app\Controllers\exim\Import.php(473): App\Models\exim\import\Import_model->get_datatablesimportjkt(Array)
#6 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(927): App\Controllers\exim\Import->Datatablesimportjkt()
#7 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Import))
#8 C:\xampp\htdocs\wiharta\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\wiharta\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2024-05-31 17:17:21 --> Unable to connect to the database.
Main connection [MySQLi]: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `beacukai_pib`.`seq` as `seq`, `aju`, `pengirim`, `HSCODE`, `penjual`, `pib`, `tglpib`, `kgm`, `desk`, `no_coo`, `tgl_coo`, `emkl_ptj`, `bl`, `negara`, `party`, `etd`, `eta`, `eta_wka`, `fasilitas`, `term`, `inv`, `MU`, `nilai_pib_usd`, `nilai_pib_idr`, `bm`, `bmt_bmad_bmi_bmtp`, `ppn`, `pph`, `beacukai_pib`.`remark`, `bm_dibebaskan`, `ppn_tidakdipungut`, `total_payment_pib`
FROM `beacukai_pib`
LEFT JOIN `beacukai_vendor` ON beacukai_vendor.`seq`=beacukai_pib.`id_vendor`
ORDER BY `kgm` ASC
 LIMIT 25', [], false)
 3 SYSTEMPATH\Model.php(772): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Models\exim\import\Import_model.php(181): CodeIgniter\Model->__call('get', [])
 5 APPPATH\Controllers\exim\Import.php(473): App\Models\exim\import\Import_model->get_datatablesimportjkt([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\exim\Import->Datatablesimportjkt()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\exim\Import))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
